sourcegraph/learn
# Record index page redesign plus load more records functionality

### What should this PR do?
Resolves [DEVED-173](https://sourcegraph.atlassian.net/browse/DEVED-173) by distinguishing internally between record index pages and tag index pages, adding **load more** functionality to both, and refreshing the design of the record index pages (currently `posts` and `videos`).

### Why are we making this change?
- We want users to have the ability to load more records upon request.
- We want our records and tag pages to have separate and meaningful designs.

### What are the acceptance criteria? 
- The **Load More** button works on tag and record pages (`/posts` and `/videos`), to load more records in cases where there are more than 10 records (the initial batch size).
- ^^ For tag pages.
- The header design on record index pages looks reasonable on desktop and mobile.
- The header on tag index pages looks the same.
- The **Load More** button is not rendered in cases where there are fewer than 10 records.
- ^ Same thing for cases where there are no other records to load.

### How should this PR be tested?
- Check out the branch, and ensure that the load more functionality works on tag pages and record index pages.
- Ensure that record index pages look reasonable on desktop and mobile.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
